templates/emdna_processing.py

```diff
@@ -11,6 +11,10 @@ def preprocess_dna(ds):
     ds = ds.rename_dims({"x": "lon", "y": "lat"})
     ds = ds.set_index(lat="latitude", lon="longitude", time="date")
     ds["time"] = [datetime.strptime(str(d), "%Y%m%d") for d in ds["time"].values]
+    ds["tasmax"] = ds["tmean"] + (
+        0.5 * ds["trange"]
+    )  # TODO: do this more miranda-like?
```

Comment:
No, I think this is fine. I think when we're calculating new variables from existing variables (something fancier than unit conversion), it falls within the scope of xclim.